archCompare/abstractArchive.py

```diff
@@ -14,6 +14,7 @@ def __init__(self, **kwargs):
         self.cmp_type = kwargs.get('cmp_type', None)
         self.remove_tmp = kwargs.get('remove_tmp', None)
         self.outfile = kwargs.get('outfile', None)
+        self.quiet = kwargs.get('quiet', None)
```

Comment:
EDIT: See my comment where you actually capture this value from the command line, that may be more appropriate.

This is confusing, here you tag this as `quiet` but it's used by a `verbose` value later.  I understand the logic of why you've done this but rather than internally saying this is quiet perhaps do:

```
self.verbose = not kwargs.get('quiet', None)
```

Then the use against verbose later will make sense.  When a variable is verb it should only do so when `True`.

Reply:
done